README.md

```diff
@@ -239,7 +271,9 @@ IE 11
 not dead
 ```
 
-We recommend using a **minifier** like [csso](https://github.com/css/csso) to compress your final compiled CSS and **sourcemaps** like [`gulp-sourcemaps`](https://www.npmjs.com/package/gulp-sourcemaps) to keep track of the location of all the source Sass for easier debugging.
+**Minfiers and sourcemaps:** We recommend using a minifier like [csso](https://github.com/css/csso) to compress your final compiled CSS and sourcemaps like [`gulp-sourcemaps`](https://www.npmjs.com/package/gulp-sourcemaps) to keep track of the location of all the source Sass for easier debugging.
```

Comment:
Typo in the word 'minifiers'. Additionally, could we break minifiers and sourcemaps into their own sections, or maybe provide a pause in the sentence to separate the thoughts? I read this first as "compress your final compiled CSS and sourcemaps". 
```suggestion
**Minifiers and sourcemaps:** We recommend using a minifier like [csso](https://github.com/css/csso) to compress your final compiled CSS and sourcemaps like [`gulp-sourcemaps`](https://www.npmjs.com/package/gulp-sourcemaps) to keep track of the location of all the source Sass for easier debugging.
```